src/coalesce-vue-vuetify3/src/components/input/c-datetime-picker.vue

```diff
@@ -23,6 +23,7 @@
   <v-text-field
     v-else
     class="c-datetime-picker"
+    :class="{ 'today-btn': showTodayButton }"
```

Comment:
This is an unusual class name. The entire component is not a `today-btn`. Only the button itself inside the component's menu would make sense to have a `today-btn` class.

Consider a more appropriate class name like `c-datetime-picker__menu--has-today-btn` on the menu's contentClass, which also adheres to the (mildly consistent) use of [BEM selectors](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/) in this project.